server/container_create_linux.go

```diff
@@ -330,17 +331,74 @@ func (s *Server) createSandboxContainer(ctx context.Context, containerID string,
 			specgen.AddMount(mnt)
 		}
 	}
-	mountLabel := sb.MountLabel()
-	processLabel := sb.ProcessLabel()
-	selinuxConfig := containerConfig.GetLinux().GetSecurityContext().GetSelinuxOptions()
-	if selinuxConfig != nil {
-		var err error
-		processLabel, mountLabel, err = getSELinuxLabels(selinuxConfig, privileged)
-		if err != nil {
+	imageSpec := containerConfig.GetImage()
```

Comment:
why all of this image stuff has been moved up/down?

Reply:
All of image stuff needed to be done before container creation, and container creation needs to be done before using the mount label.  I could move it back and then move the calls that use the mount label to be after the container creation.